python/bindings/openravepy_ikparameterization.cpp

```diff
@@ -597,6 +600,7 @@ void init_openravepy_ikparameterization()
                                    .def("__repr__",&PyIkParameterization::__repr__)
                                    .def("__mul__",&PyIkParameterization::__mul__)
                                    .def("__rmul__",&PyIkParameterization::__rmul__)
+                                   .def("toDict",&PyIkParameterization::toDict)
```

Comment:
@yoshikikanemoto is `toDict` some python reserved word? If not, why not just call it `SerializeJSON`?

Reply:
toDict is the function name that ujson tries to use when a given object is not compatible with python native types.

Reply:
toDict is the function name that ujson tries to use when a given object is not compatible with python native types.

Reply:
discussed with rosen. we better explicitly call `SerializeJSON` when we serialize JSON from openrave classes since conversion is not symmetric and we might not continue using ujson forever. then close this MR